balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -85,10 +76,6 @@ class MetricsServlet extends JacksonJsonServlet
       contentType = response.contentType
       response.result
     }
-    finally
-    {
-      periodMeter.update(System.currentTimeMillis() - begin, TimeUnit.MILLISECONDS)
```

Comment:
I looked at the link and searched around. I can't seem to find an example of someone using a timer in the way we were using the yammer one, that is to say, calling an update method with a provided time. Adding the MetricsSupport trait gives you the ability to specify a block that gets run on a timer: ```timer("timer") { // code here }```, but that's not quite what we want. I'm going to check into the dropwizard bits directly seeing as they're already used elsewhere.